### PR TITLE
Contact me form added to the portfolio project

### PR DESCRIPTION
**Pull request by kkolade**
The contact form section was added to the project and style as close to the template as possible
While executing the project, the following objectives were adhered to:

- Name field accepts up to 30 chars.
- Email field accepts only emails.
- Message field accepts up to 500 chars.
- All fields have placeholder text.
- All fields have the correct type.
- A user cannot submit the form if one or more of the fields is empty.
- Data collection by using the [Formspree service](https://formspree.io/html).
- Flexbox is used In order to lay out the elements on the page.
- Button interactions implemented as per the template(enable, hover, etc.).

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -237,6 +237,52 @@ img {
...
+  align-items: center;
+}
+
+.contact__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding-bottom: 8px;
+}
```
Comment (jaflih) [inline css/style.css:255]: Please add a backgroud image for Desktop and Mobile  view to stick to Figma design.

Figma: 
image


Your design
Capture d’écran 2022-11-02 à 00 09 01

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @kkolade , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
Correct Gitflow has been followed ✔️
Descriptive README.md file ✔️
Pull Request has a title and summary ✔️
No Linter error ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @jaflih in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
Images added to the background of the 'Hero' and 'Contact' as per the Figma template

File: index.html [hunk 2]
```diff
@@ -296,5 +296,52 @@ <h1 class="about__headline headline-text">About <br />Myself</h1>
...
+          ></textarea>
+          <input
+            type="submit"
+            value="Get in Touch"
+            class="contact__cta btn-primary"
+
+          />
+         
+        </form>
+        <div class="bottom_line"></div>
```
Comment (aimalamiri) [inline index.html:343]: Good job, but it's highly recommended to follow the exact Figma template, so please make sure to remove the extra space under the bottom line.

image

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @kkolade, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

AlmostThereReachGIF


 ## Highlights

- The form is designed very well and looks fantastic ✔️ ⭐
- Good job for using the right GitHub Flow 💯
- Awesome file organization 🗃️
- Commits are descriptive and meaningful 👍
- Linters are properly set up and passing 🟢 👍

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
The extra space at the bottom of the contact section has been removed and the project is more reflective of the Figma template

**Review by anagudelogu — APPROVED** [review]
Hi @kkolade, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Great use of GitHub flow. 👍🏼 
✅ Great UI design! 💯 
✅ Descriptive commit messages. 💬 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**